release/angular-ui-router.js

```diff
@@ -109,7 +109,7 @@ function inheritParams(currentParams, newParams, $current, $to) {
   var parents = ancestors($current, $to), parentParams, inherited = {}, inheritList = [];
 
   for (var i in parents) {
-    if (!parents[i].params) continue;
+    if (!parents[i] || !parents[i].params) continue;
```

Comment:
[hasOwnProperty](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty) might be a better check. Also, the correct place to do this fix is at https://github.com/angular-ui/ui-router/blob/master/src/common.js#L99